app/v1/applications/sql.js

```diff
@@ -245,8 +245,19 @@ function getAppPermissionsId (id) {
         }).toString();
 }
 
+function getCategoryByName(name) {
+    return sql.select('id', 'name', 'display_name')
+        .from('categories')
+        .where(
+            {
+                'name': name
+            }
+        )
+        .toString();
+}
+
 function getAppCategory (id) {
-    return sql.select('categories.id', 'display_name')
+    return sql.select('categories.id','display_name')
```

Comment:
Unnecessary deletion.

```suggestion
    return sql.select('categories.id', 'display_name')
```